src/mo_values/prim.ml

```diff
@@ -321,6 +321,18 @@ let prim trap =
       in go (fun xs -> xs) k 0
     | _ -> assert false
     )
+  | "Array.tabulateVar" -> fun c v k ->
```

Comment:
Err, let's add a todo to optimize both of these - they are way more expensive than they should be... As is representing an array by an array of refs..